src/models/TableModel/impl/InMemoryTable.ts

```diff
@@ -8,11 +8,13 @@ import { ValueTypeName } from '../ValueTypeName'
 import { CxValue } from '../../CxModel/Cx2/CxValue'
 import { AttributeDeclaration } from '../../CxModel/Cx2/CoreAspects/AttributeDeclarations'
 import { translateCXEdgeId } from '../../NetworkModel/impl/CyNetwork'
-export const createTable = (id: IdType): Table => ({
-  id,
-  columns: [],
-  rows: new Map<IdType, Record<AttributeName, ValueType>>(),
-})
+
+export const createTable = (id: IdType, colNames: Column[] = []): Table => (
```

Comment:
colNames should be renamed to `columns` if these are actually column objects.

Reply:
okay